docs/motivation/evaluation-of-robotics-data-recording-file-formats.md

```diff
@@ -1,5 +1,7 @@
 # Evaluation of robotics data recording file formats
 
+<!-- cspell:words packetized pluggable checksumming mitigations Karsten Knese Hurliman jhurliman -->
+
 _Oct 6, 2021_  
 _John Hurliman (@jhurliman)_  
 _Foxglove Technologies Inc_
```

Comment:
we could link these to @jhurliman and foxglove.dev?

Reply:
good point, I thought github would auto-link it